Exercise01/Model/CreateData.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Exercise01.Model
+{
+    class CreateData

```

Comment:
Never name a class starts with a verb. It should be `StudentGenerator` or `StudentFactory`

Reply:
Done